doc/lua_api.txt

```diff
@@ -2201,13 +2201,15 @@ appearing when you don't expect them to. See `no_prepend[]`
 
 #### `textarea[<X>,<Y>;<W>,<H>;<name>;<label>;<default>]`
 * Same as fields above, but with multi-line input
-* if the text overflows a vertical scrollbar is added
-* if the name is empty the textarea is readonly, the label is not displayed.
+* If the text overflows, a vertical scrollbar is added.
+* If the name is empty, the textarea is readonly and
+  the background is not shown.
 
 #### `label[<X>,<Y>;<label>]`
 * `x` and `y` work as per field
 * `label` is the text on the label
 * Position and size units are inventory slots
+* If you want to show text with multiple lines, use textarea with empty name.
```

Comment:
More concise option: "To show text with ... "
But it's better to list things under what you are talking about. If you need to explain usage it should be "Use for single line text labels" and then move this to textarea section.

Reply:
I've just noticed that it's not said what label actually does: It displays text.

Reply:
You've basically ignored my comment. Don't talk about textarea in label section. Talk about textarea in textarea section.